tests/src/test/java/io/streamnative/pulsar/handlers/kop/KopEventManagerTest.java

```diff
@@ -74,7 +74,7 @@ protected void cleanup() throws Exception {
         super.internalCleanup();
     }
 
-    @Test(timeOut = 6000)
+    @Test(timeOut = 60000000)
```

Comment:
Why timeout 60000s ?

Reply:
it is a leftover, when I used the debugger/breakpoints.
Fixed